reference/release-notes/1.23.1.md

```diff
@@ -0,0 +1,76 @@
+---
+orphan: true
+---
+# 1.23.1
+
+These release notes cover new features and changes in Anbox Cloud 1.23.1.
+
+Anbox Cloud 1.23.1 is a patch release. To understand minor and patch releases, see {ref}`ref-release-notes`.
+
+Please see {ref}`ref-component-versions` for a list of updated components.
+
+## Requirements
+
+See {ref}`ref-requirements` for detailed requirements to run Anbox Cloud 1.23.1.
+
+## New features & improvements
+
+### Anbox Cloud Appliance (epoch=1)
+
+* You can now change the appliance network configuration using `anbox-cloud-appliance config set`. You can change the public IP address and location of the appliance.<!--AC-2727-->.
+* You can now configure the [CORS settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) for those API endpoints that the appliance reverse proxy passes to the Anbox Stream Gateway. You can set the following using the `anbox-cloud-appliance config set` command:<!--AC-2732-->
+  - The network's public IP address (`network.public_address`)
+  - The network's DNS name (`network.location`)
+  - The [HTTP origin](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin) (`core.https_allowed_origin`)
+  - List of allowed HTTP headers (`core.https_allowed_headers`)
+  - List of allowed HTTP methods (`core.https_allowed_methods`)
+* You can skip setting up coturn when initialising the appliance and can configure a custom STUN server.<!--AC-2719-->
+* The [`prepare-node-script`](prepare-node-script.md) command is extended to support systems with an already installed NVIDIA driver. <!--AC-2714-->
```

Comment:
We miss (maybe I forgot to have a JIRA story for this):

* You can now set up a custom identity provide for the UI by configuring an OpenID Connect provider through the preseed configuration at initialization time